str_analysis/check_combined_results_tsv_for_pathogenic_repeats.py

```diff
@@ -0,0 +1,274 @@
+"""This script makes it easier to manually look through genotypes of many samples at known pathogenic loci and
```

Comment:
when is this script run and who is the output for? is this just for a quick validity check?